package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "@edx/edx-bootstrap": "0.4.3",
     "@edx/paragon": "1.7.1",
-    "@edx/studio-frontend": "1.2.5",
+    "@edx/studio-frontend": "git://github.com/edx/studio-frontend.git#efischer/export_i18n",
```

Comment:
Fun fact: If you install SFE into a sandbox using a git link, you'll need to un-gitignore `dist/` like this: https://github.com/edx/studio-frontend/pull/124/commits/21de7e44771d69672e82198af0742418b4f44c1c